src/Resources/app/administration/src/module/swag-paypal-payment/component/swag-paypal-payment-details-v2/index.js

```diff
@@ -169,31 +169,17 @@ Component.register('swag-paypal-payment-details-v2', {
 
         getTransactionFee(type, payment) {
             if (type === 'capture') {
-                const sellerReceivableBreakdown = payment.seller_receivable_breakdown;
-                if (sellerReceivableBreakdown === null) {
-                    return null;
+                const paypalFee = payment.seller_receivable_breakdown?.paypal_fee;
+                if (paypalFee) {
+                    return `${paypalFee.value} ${paypalFee.currency_code}`;
                 }
-
-                const paypalFee = sellerReceivableBreakdown.paypal_fee;
-                if (paypalFee == null) {
-                    return null;
-                }
-
-                return `${paypalFee.value} ${paypalFee.currency_code}`;
             }
 
             if (type === 'refund') {
-                const sellerPayableBreakdown = payment.seller_payable_breakdown;
-                if (sellerPayableBreakdown === null) {
-                    return null;
+                const paypalFee = payment.seller_payable_breakdown?.paypal_fee;
+                if (paypalFee) {
+                    return `${paypalFee.value} ${paypalFee.currency_code}`;
                 }
```

Comment:
that looks way too identical, couldn't we combine the IF?

Reply:
sure, but only for a small benefit, as seller\_**payable**\_breakdown and seller\_**receivable**\_breakdown are not the same

Reply:
never mind, I was too blind to see the difference.